desktop/Program.fs

```diff
@@ -191,6 +191,8 @@ let main _ =
   let mutable options = WindowOptions.Default
   options.Size <- Vector2D<int>(screenWidth * zoom,screenHeight * zoom)
   options.Title <- "F# Wolfenstein"
+  Silk.NET.Input.Sdl.SdlInput.RegisterPlatform()
```

Comment:
That's manual registration of platforms. By default Silk.Net uses dynamic platform registration, and this is not working well with NativeAOT, so registering platforms manually.